addon/ng2/commands/github-pages-deploy.ts

```diff
@@ -52,8 +53,16 @@ module.exports = Command.extend({
       type: String,
       default: '',
       description: 'Github username'
+    }, {
+      name: 'output-path',
+      type: String,
+      default: 'dist/',
```

Comment:
Too many flags will be confusing to new users using `ng --help`. I say having it in the config is enough. I do think it's unnecessary.